src/scanpy/_utils/__init__.py

```diff
@@ -532,6 +533,19 @@ def update_params(
     return updated_params
 
 
+# `get_args` returns `tuple[Any]` so I don’t think it’s possible to get the correct type here
+def get_literal_vals(typ: UnionType | Any) -> KeysView[Any]:
```

Comment:
Why make this return `KeysView`? I would think a `List` would be a more natural return a list

Reply:
Or even a set.  But forcing keysview is a strong choice

Reply:
`KeysView` is “ordered set”